packages/cli/package.json

```diff
@@ -25,7 +25,7 @@
     "@heroku-cli/schema": "^1.0.25",
     "@heroku/buildpack-registry": "^1.0.1",
     "@heroku/eventsource": "^1.0.7",
-    "@oclif/core": "^2.8.11",
+    "@oclif/core": "^2.16.0",
```

Comment:
Somewhat significant jump in `oclif/core` but I don't see much in the changelog that we should be worried about. I noticed 2.16 was released literally yesterday (doesn't even show up in the changelog yet, heh). I think you had described to me something that the oclif team fixed that you asked about. Is that the update with 2.16?

Reply:
Yes. They released a fix for us that allows for hidden aliases. We need it for the v9 release as well, so this was really helpful.